blog/konferencja-tlumaczy-2024/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Konferencja tłumaczy 2024 - patronujemy!'
+authors: niedzwiedzka
+date: '2024-06-25'
+tags:
+  - 'news'
+  - 'z-kraju'
+  - 'konferencje'
+  - 'tlumaczenia'
+---
+Kolejny rok i kolejna [Konferencja Tłumaczy (KTLC)](https://konferencjatlumaczy.pl/). Już od kilku lat jesteśmy jej dumnym patronem - nie inaczej jest w tym roku. I nie tylko patronem - uczestniczymy w konferencji (czasami nawet jako prelegenci i prelegentki) i możemy ją wam polecić z absolutnym spokojem o wysoką merytoryczną jakość oraz świetną atmosferę. 
+
+<!--truncate-->
+
+#### **Gdzie i kiedy?**

```

Comment:
Czy te nagłówki nie powinny lecieć kolejno? Tytuł to H1, więc kolejny powinien być H2. 
Poza tym H2 i H3 są pokazane też w tym mini toc po prawej stronie, H4 już tam chyba nie trafia? 

![image](https://github.com/techwriter-pl/techwriter-pl.github.io/assets/119131560/b0ba825e-eedf-48b3-b8a9-be227cad2ae8)

Reply:
Zgadza się, nagłówki sekcji powinny być H2.

Reply:
Wzorowałam się na innym tekście, gdzie były H4 (ale już poprawiam, dzięki).

Reply:
Prawdopodobnie trafiłaś na jakiś artykuł, który akurat był złym przykładem :(